### ISSUE TITLE
Merriweather 700 italic fails validation

### ISSUE THREAD

**Issue by astoilkov** [google/fonts#1599]
Running validation on Merriweather 700 italic it fails with `ERROR | The "head" table checkSumAdjustment (0x49a12a1c) does not match the calculated checkSumAdjustment (0x8dfcfa15).`.

This prevents us from using `opentype.js` to work with the font data. See the [issue here](https://github.com/nodebox/opentype.js/issues/349).

[Merriweather-700-italic.woff.zip](https://github.com/google/fonts/files/2110621/Merriweather-700-italic.woff.zip)

**Comment by m4rc1e** [comment]
Do you get the same issue using the official download, https://fonts.google.com/specimen/Merriweather?

We don't provide woffs to download, only ttfs.

**Comment by astoilkov** [comment]
Yes. If you navigate the UI you can only get the ttfs. However, if I request the `https://fonts.googleapis.com/css?family=Merriweather` URL with the correct agent string I will get URL's for `.woff` files.

How do you create the `.woff` files?

**Comment by astoilkov** [comment]
`.ttf` files work corrently when using `opentype.js`. However, we want to use `.woff` for optimizing disk space our app uses.
